packages/shared/src/components/Feed.tsx

```diff
@@ -185,7 +185,10 @@ export default function Feed<T>({
 
   const infiniteScrollRef = useFeedInfiniteScroll({
     fetchPage,
-    canFetchMore: canFetchMore && !showScrollOnboardingVersion,
+    canFetchMore:
+      canFetchMore &&
+      !showScrollOnboardingVersion &&
+      feedQueryKey?.[0] !== RequestKey.FeedPreview,
```

Comment:
Can't you simply use `feedName` here?
It's not super clear what the first item on a feedQuery is, and can be optional even.

Reply:
In feeds its standardized and I wanted to use the same logic as useFeedPreviewState to avoid any issues down the line. Its a flaster until lean feed either way.